requirements.txt

```diff
@@ -1,11 +1,7 @@
 praw==4.4.0
 redis<3.0.0
-hiredis
```

Comment:
Perhaps I haven't gotten there yet, but even though we don't call this library, it's used as a speedup by the redis library.